internal/stack/resources.go

```diff
@@ -171,6 +184,45 @@ func applyResources(profile *profile.Profile, stackVersion string) error {
 	return nil
 }
 
+func addClientCertsToResources(resourceManager *resource.Manager, certResources []resource.Resource) error {
```

Comment:
Could we use an approach like this one for serverless? So we provide the certificates when creating the output and we don't need to update it later?

Reply:
But for serverless the certificates are created at a later stage than adding the output to agent policy. And to add agent policy the output needs to be created

Reply:
I think we could move `applyServerlessResources` to an early stage if needed. It is only now in `startLocalServices` because so far it was only needed for local services. But we could run this earlier if needed to create the project.